src/split/src/Split.tsx

```diff
@@ -28,6 +29,10 @@ export const splitProps = {
     type: Number,
     default: 0.5
   },
+  size: {
+    type: Number,
+    default: 0
```

Comment:
not sure if this should default to 0. this should default to undefined so default-size will work and 0 as value should still be possible